torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12340,6 +12340,20 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_meta_outplace', device_type='cuda'),
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_symbolic_meta', device_type='cuda'),
            )),
+    OpInfo('view_copy',
+           dtypes=all_types_and(torch.bool, torch.bfloat16, torch.float16),
+           ref=lambda x, newshape: np.reshape(x, newshape).copy(),
+           supports_out=True,
+           supports_forward_ad=True,
+           supports_fwgrad_bwgrad=True,
+           supports_autograd=True,
+           sample_inputs_func=sample_inputs_view_reshape,
+           error_inputs_func=error_inputs_view_reshape,
+           skips=(
+               # The size of tensor a (0) must match the size of tensor b (5) at non-singleton dimension 1
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out'),
```

Comment:
I wonder what's going on here? Do we need an issue that this comment can point to?

Reply:
Yes, please file an issue and cite it in the comment. Tensors with zero elements should be shaped to the correct shape. See https://github.com/pytorch/pytorch/wiki/Developer-FAQ#how-does-out-work-in-pytorch

Reply:
added in https://github.com/pytorch/pytorch/issues/89068